modules/oadp-installing-dpa-1-2-and-earlier.adoc

```diff
@@ -113,35 +121,38 @@ spec:
     velero:
       defaultPlugins:
         - azure
-        - openshift <1>
-      resourceTimeout: 10m <2>
+        - openshift # <1>
+      resourceTimeout: 10m # <2>
     restic:
       enable: true <3>
       podConfig:
-        nodeSelector: <node_selector> <4>
+        nodeSelector: <node_selector> # <4>
   backupLocations:
     - velero:
         config:
-          resourceGroup: <azure_resource_group> <5>
-          storageAccount: <azure_storage_account_id> <6>
-          subscriptionId: <azure_subscription_id> <7>
+          resourceGroup: <azure_resource_group> # <5>
+          storageAccount: <azure_storage_account_id> # <6>
+          subscriptionId: <azure_subscription_id> # <7>
           storageAccountKeyEnvVar: AZURE_STORAGE_ACCOUNT_ACCESS_KEY
         credential:
           key: cloud
-          name: {credentials}  <8>
+          name: {credentials}  # <8>
         provider: {provider}
         default: true
         objectStorage:
-          bucket: <bucket_name> <9>
-          prefix: <prefix> <10>
-  snapshotLocations: <11>
+          bucket: <bucket_name> # <9>
+          prefix: <prefix> # <10>
+  snapshotLocations: # <11>
     - velero:
         config:
           resourceGroup: <azure_resource_group>
           subscriptionId: <azure_subscription_id>
           incremental: "true"
         name: default
```

Comment:
can you remove the "name" key value pair from here as it's not supported for VSL

Reply:
@shdeshpa07 - please can you then go back and check what has been published to 4.14, 4.15, 4.16
It looks like this has been published

https://docs.openshift.com/container-platform/4.15/backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure.html#oadp-installing-dpa-1-2-and-earlier_installing-oadp-azure

![image](https://github.com/user-attachments/assets/4e29df38-f748-4eb7-9abb-91a8f9b1c318)

https://docs.openshift.com/container-platform/4.15/backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure.html#oadp-installing-dpa-1-3_installing-oadp-azure

![image](https://github.com/user-attachments/assets/9ca4715c-cfa1-45d2-8396-58841acb11ed)

Reply:
Replied to this comment in PR - https://github.com/openshift/openshift-docs/pull/79643